<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_Name">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>28</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>İsim</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Password">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>32</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Şifre</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Login">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Giriş Yap</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Name">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Password">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBox_Password_Visible">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Şifreyi Göster</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
